img/l/line6.com.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1133.858" height="272.48" version="1.0">
+   <path d="M1188.89-768.727c13.092.813 229.772 14.692 476.484 15.95 246.711-1.257 463.4-15.137 476.492-15.95 26.012-2.832 32.093-14.366 32.093-32.093v-105.64c0-17.727-6.081-29.26-32.093-32.093-13.092-.813-229.772-14.693-476.484-15.95-246.711 1.257-463.4 15.137-476.492 15.95-26.011 2.832-32.093 14.366-32.093 32.093v105.64c0 17.727 6.082 29.26 32.093 32.093zm900.56-70.384c-.361-7.828-16.672-11.904-36.715-11.374-20.046.525-35.94 5.444-35.574 13.272.361 7.829 16.655 11.569 36.702 11.044 20.042-.53 35.953-5.117 35.587-12.942zm29.03-55.126c2.629-.069-13.132 21.858-13.132 21.858s-17.232-9.717-45.163-8.986c-29.902.788-53.22 8.737-52.802 24.902 16.75-7.777 32.734-10.7 54.42-11.272 34.503-.904 70.788 8.143 71.714 28.13.985 21.32-41.347 30.771-76.504 31.696-55.535 1.46-94.609-11.021-96.012-41.333-1.282-27.647 41.91-46.958 94.157-48.33 20.701-.547 34.886.25 44.802 1.32 9.925 1.072 15.562 2.092 18.52 2.015zm-251.705 69.47h-96.88v-18.841h90.378v-18.024h-90.379v-16.733h95.255l-3.882-17.904h-149.112v89.406h152.305l2.315-17.903zm-195.901 22.096 17.89-4.274v-89.405h-53.947v48.916l-101.447-52.953-17.473 4.037v89.405h53.947v-47.276l101.03 51.55zm-284.111-75.66h25.414v53.367h-25.414l-2.445 18.02h113.696l-2.444-18.02h-25.654v-53.367h25.654l2.444-18.02h-113.696l2.445 18.02zm-19.405 71.386v-24.85l-45.521 6.947h-46.601v-71.502h-57.739v89.405h149.86zm534.502 38.196c-70.47 2.384-151.651 4.33-236.465 4.786-245.523-1.313-451.315-15.128-464.36-15.937-26.166-2.208-32.088-4.015-32.088-32.093v-83.294c-.246-20.86 1.166-30.776 32.089-32.093 13.044-.81 218.836-14.624 464.36-15.937 85.101.456 166.54 2.414 237.178 4.812 20.163 54.265 18.23 115.409-.714 169.756z" style="fill:#000" transform="translate(-1111.364 989.88)"/>
+</svg>

```

Comment:
Here's an inverted version of the one you provided. It seems it's the version used on their website.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1017.2 201.7"><path d="M32 185.8c13.2.8 229.9 14.7 476.6 16 246.7-1.3 463.4-15.2 476.5-16 26-2.9 32-14.4 32-32.1V48c0-17.7-6-29.2-32-32C972 15 755.3 1.3 508.6 0A8823.8 8823.8 0 0 0 32 16C6 18.8 0 30.3 0 48v105.7c0 17.7 6 29.2 32 32z"/><path fill="#fff" d="M32 185.8c13.2.8 229.9 14.7 476.6 16 246.7-1.3 463.4-15.2 476.5-16 26-2.9 32-14.4 32-32.1V48c0-17.7-6-29.2-32-32C972 15 755.3 1.3 508.6 0A8823.8 8823.8 0 0 0 32 16C6 18.8 0 30.3 0 48v105.7c0 17.7 6 29.2 32 32zm900.7-70.4c-.4-7.8-16.7-12-36.8-11.4-20 .5-35.9 5.5-35.5 13.3.3 7.8 16.6 11.6 36.7 11 20-.5 36-5 35.6-13zm29-55.1c2.6-.1-13.1 21.8-13.1 21.8s-17.3-9.7-45.2-9c-30 .8-53.2 8.8-52.8 25A129 129 0 0 1 905 86.7c34.5-1 70.8 8.1 71.7 28.1 1 21.3-41.3 30.8-76.5 31.7-55.5 1.5-94.6-11-96-41.3-1.3-27.7 42-47 94.2-48.4 20.7-.5 34.8.3 44.8 1.4 9.9 1 15.5 2 18.5 2zM710 129.7h-97V111h90.4V93H613V76h95.3l-4-17.9h-149v89.4h152.3l2.3-17.9zm-196 22.1 18-4.2V58.2h-54V107L376.6 54l-17.5 4v89.5h54v-47.3zM230 76.2h25.4v53.3H230l-2.5 18h113.7l-2.4-18H313V76.2h25.7l2.4-18H227.5zm-19.4 71.4v-24.9l-45.6 7h-46.6V58.2H60.7v89.4h149.9zM745 185.8a8524 8524 0 0 1-236.5 4.7c-245.5-1.3-451.3-15-464.4-15.9-26.1-2.2-32-4-32-32V59.1c-.3-20.8 1.1-30.8 32-32 13-.9 218.9-14.7 464.4-16 85.1.4 166.5 2.4 237.2 4.8 20.1 54.3 18.2 115.4-.7 169.8z"/></svg>
```